WORKSPACE

```diff
@@ -108,41 +101,32 @@ http_archive(
 #    build_file = "third_party/libtorch/BUILD"
 #)
 
-#new_local_repository(
-#    name = "cudnn",
-#    path = "/usr/",
-#    build_file = "@//third_party/cudnn/local:BUILD"
-#)
+new_local_repository(
```

Comment:
Same thing here, we have no idea what versions of the libraries are installed on peoples systems and TRTorch does crash if you use the wrong ones. Thats why we suggest people download the tars and use `dist-dir`. If they know it is correct or on Jetson were we can ensure the environment is uniform then this is fine

Reply:
Basically I'd rather people get a error from bazel that the system is not what was expected and them either modifying the WORKSPACE to point to what they have or realizing they dont have the right dependencies than a bunch of bug reports saying that people are getting runtime crashes because they inadvertently linked the wrong cuDNN.

Reply:
(Also I can share with you a WORKSPACE file which pulls at compile time from the build server for cuDNN and TensorRT, which makes sure you always have the right versions)

Reply:
Reverted local/archive changes.